lib/tasks/docker.rake

```diff
@@ -14,6 +14,8 @@
 # Other useful environment variables (not specific to this rake task)
 # => COMMIT_HASH    used by the discourse_test docker image to load a specific commit of discourse
 #                   this can also be set to a branch, e.g. "origin/tests-passed"
+# => SKIP_LINT      if using the discourse_test docker image, this will skip the docker:test task, and only run the docker:lint task
```

Comment:
It would be tidier, but I worry it could cause issues. @SamSaffron was there a reason it was refactored out of the rake task in https://github.com/discourse/discourse/commit/7289e0b5fd0aafd0fd8004be373e6487d76fdbbe ?

Maybe something to do with trying to do `git pull` while `rake` is running? I doubt it would go well trying to update files that are currently open... Or maybe that dependencies need to be installed **after** `git pull`, but before running the rake task...